packages/core/saved-objects/core-saved-objects-migration-server-internal/src/document_migrator/model_version.ts

```diff
@@ -22,6 +22,8 @@ import {
 import { TransformSavedObjectDocumentError } from '../core';
 import { type Transform, type TransformFn, TransformType, type TypeVersionSchema } from './types';
 
+// @TINA this is how we extract the schemas from a model_version for the document migrator
+// Question: we need this as a helper, can we move it to core-saved-objects-api-server-internal.src.lib/apis/helpers/validation ??
```

Comment:
FWIW, This is not the same schema you'll want to use. 

This method extracts the `forwardCompatibility` schema, which is not the same as the `create` schema.

Reply:
ya, sorry, forgot to delete that comment